app/src/main/java/org/merakilearn/datasource/network/model/Classes.kt

```diff
@@ -50,5 +52,5 @@ data class Classes(
 }
 
 fun Classes.sanitizedType(): String = type.replace("_", " ").capitalizeWords()
-
-fun Classes.timeRange(): String = "${startTime.toTime()} - ${endTime.toTime()}"
\ No newline at end of file
+fun Classes.timeRange(): String = "${startTime.toTime()} - ${endTime.toTime()}"
+fun Classes.dateFormatUI(): String = "${startTime.toDate()}"
```

Comment:
Knitpicking on this one but UI is not a very intuitive name, something like formattedDate or displayableDate would make more sense. Feel free to ignore this one